samples/unit-tests/series/datagrouping/demo.js

```diff
@@ -435,6 +435,12 @@ QUnit.test('Switch from grouped to non-grouped', function (assert) {
         'Monthly columns'
     );
 
+    const dataGrouping = chart.addSeries({}).options.dataGrouping;
+    assert.ok(
+        dataGrouping.forced && dataGrouping.units,
+        '#15512: Datagrouping from range selector should be used'
```

Comment:
Should we rather check if options are exactly the same? Something like `assert.deepEqual(dataGrouping, rangeSelector.buttonOptions[rangeSelector.selected].dataGrouping)`?